packages/lib/RotatingLogs.test.ts

```diff
@@ -0,0 +1,51 @@
+import { open, close, write, readdir, rm } from 'fs-extra';
+import RotatingLogs from './RotatingLogs';
+import { createTempDir } from './testing/test-utils';
+
+jest.useFakeTimers();
+
+const createEmptyFileOfSize = async (fileName: string, size: number) => {
+	const fd = await open(fileName, 'w');
+	await write(fd, 'test', size);
+	await close(fd);
+};
+
+describe('RotatingLogs', () => {
+	let dir: string;
+	let files: string[];
+	let rotatingLogs: RotatingLogs;
+
+	beforeEach(async () => {
+		dir = await createTempDir();
+		rotatingLogs = new RotatingLogs(dir, 1024, 1000);
+		await createEmptyFileOfSize(`${dir}/log.txt`, 1024);
+	});
+
+	afterEach(async () => {
+		files = [];
+		await rm(dir, { recursive: true });
```

Comment:
Somehow I can't find any reference to "rm" in the fs-extra doc, is that an alias for "remove"? But "remove" doesn't have this extra parameter (because it's recursive by default). Could you switch to use "remove" please?